contrib/hack/test_walkthrough.sh

```diff
@@ -210,7 +210,7 @@ API_SERVER_HOST="$(kubectl get services -n catalog | grep 'apiserver' | awk '{pr
 export KUBECONFIG="${SC_KUBECONFIG}"
 
 kubectl config set-credentials service-catalog-creds --username=admin --password=admin
-kubectl config set-cluster service-catalog-cluster --server="http://${API_SERVER_HOST}:80"
+kubectl config set-cluster service-catalog-cluster --server="https://${API_SERVER_HOST}:443" --insecure-skip-tls-verify=true
```

Comment:
We should update the walkthrough docs the same way, if we want to keep it this way for now.